# DriveInfo.msg

Header header
std_msgs/Float32 direction
std_msgs/Float32 distance
std_msgs/Float32 orientation
